aalto/aaltodata.rst

```diff
@@ -126,9 +126,12 @@ everyone in the group has access.
       there are fixed multi-year projects, they can also get a group.
 
 -  Groups are managed by IT staff. To request a group, mail us with
-   the necessary information (see next point).
+   the necessary information (see bottom of page).
 -  Each group has an owner, quota on filesystems, and some other
-   metadata (see below),
+   metadata (see below).
+-  Group membership is per-account, not tied to employment contracts.
+   If you want someone to lose access, you have to explicitly remove
+   them when done.
```

Comment:
I made it explicit that they need to request for a removal (unless at CS since there group owners can do it themselves)